src/main/java/org/logstash/beats/BeatsHandler.java

```diff
@@ -7,23 +7,28 @@
 
 import java.io.IOException;
 import java.net.InetSocketAddress;
+import java.util.Objects;
 import javax.net.ssl.SSLHandshakeException;
 
 public class BeatsHandler extends SimpleChannelInboundHandler<Batch> {
-    private final static Logger logger = LogManager.getLogger(BeatsHandler.class);
+
+    private final static Logger LOGGER = LogManager.getLogger(BeatsHandler.class);
```

Comment:
why the change since they are not global variables? Seems that this are not relevant to the actual changes needed?